src/app/Sankalan/Quiz/response.php

```diff
@@ -0,0 +1,44 @@
+<?php

```

Comment:
lets split this file into two files.
1. stores responses from user.
2. the one the returns user's responses should actually a part of "get problems"